packages/kit/src/core/generate_manifest/index.js

```diff
@@ -44,7 +44,7 @@ export function generate_manifest({ build_data, relative_path, routes }) {
 	);
 
 	/** @type {(path: string) => string} */
-	const loader = (path) => `() => import('${path}')`;
+	const loader = (path) => `__memo(() => import('${path}'))`;
```

Comment:
Small comment maybe why we memo the import

Reply:
Added a comment near the __memo declaration below. That should suffice.